aiida_orca/parsers/__init__.py

```diff
@@ -69,22 +69,21 @@ def _remove_nan(parsed_dictionary: dict) -> dict:
 
         output_dict = _remove_nan(parsed_dict)
 
-        keywords = output_dict['metadata']['keywords']
+        # keywords = output_dict['metadata']['keywords']
 
         #opt_pattern = re.compile('(GDIIS-)?[CZ?OPT]', re.IGNORECASE)
 
         #if any(re.match(opt_pattern, keyword) for keyword in keywords):
         #opt_run = True
-        opt_run = False
-        for keyword in keywords:
-            if 'opt' in keyword.lower():
-                opt_run = True
-
-        if opt_run:
-            relaxed_structure = StructureData(
-                pymatgen_molecule=Molecule.
-                from_file(os.path.join(out_folder._repository._get_base_folder().abspath, fname_relaxed))  #pylint: disable=protected-access
-            )
+        # opt_run = False
+        # for keyword in keywords:
+        #     if 'opt' in keyword.lower():
+        #         opt_run = True
+
+        # if opt_run:
+        if parsed_dict['optdone']:
```

Comment:
This throws a KeyError for single point calculations. 
```python
 | [33|OrcaCalculation|on_except]: Traceback (most recent call last):
 |   File "/opt/conda/lib/python3.7/site-packages/plumpy/process_states.py", line 231, in execute
 |     result = self.run_fn(*self.args, **self.kwargs)
 |   File "/home/aiida/.local/lib/python3.7/site-packages/aiida/engine/processes/calcjobs/calcjob.py", line 388, in parse
 |     exit_code_retrieved = self.parse_retrieved_output(retrieved_temporary_folder)
 |   File "/home/aiida/.local/lib/python3.7/site-packages/aiida/engine/processes/calcjobs/calcjob.py", line 468, in parse_retrieved_output
 |     exit_code = parser.parse(**parse_kwargs)
 |   File "/home/aiida/.local/lib/python3.7/site-packages/aiida_orca/parsers/__init__.py", line 84, in parse
 |     if parsed_dict['optdone']:
 | KeyError: 'optdone'
```

I tested that the following works:
```suggestion
        if parsed_dict.get('optdone'):
```